packages/console/src/components/RadioGroup/Radio.module.scss

```diff
@@ -118,6 +118,7 @@
 
   .content {
     padding: _.unit(5);
+    height: 100%;
 
```

Comment:
Should we also add `whitespace: nowrap`?

Reply:
We support wrapping.
<img width="453" alt="image" src="https://user-images.githubusercontent.com/10806653/225205215-7d571ef5-398f-44d8-a060-1e3a4b2923ce.png">